configs.js

```diff
@@ -0,0 +1,42 @@
+let alert = document.getElementById('alert');
```

Comment:
I think this could be a const.

```suggestion
const alert = document.getElementById('alert');
```